### PR TITLE
[OSDEV-1053] Claims. Updating auto email responses in light of claims changes.

### PR DESCRIPTION
**Pull request by mazursasha1990**
[OSDEV-1053](https://opensupplyhub.atlassian.net/browse/OSDEV-1053) - Updated email notification about the claim submission.

[OSDEV-1053]: https://opensupplyhub.atlassian.net/browse/OSDEV-1053?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by roman-stolar — APPROVED** [review]
LGTM

File: doc/release/RELEASE-NOTES.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * [OSDEV-1105](https://opensupplyhub.atlassian.net/browse/OSDEV-1105) - Contribution. Allow commas in list name and update error message.
 * [OSDEV-272](https://opensupplyhub.atlassian.net/browse/OSDEV-272) - Facility Claims Page. Implement ascending/descending and alphabetic sort on FE. Applied proper sorting for lower case/upper case/accented strings.
 * [OSDEV-1036](https://opensupplyhub.atlassian.net/browse/OSDEV-1036) - Claims. Add a sortable "claim decision" column to claims admin page.
+* [OSDEV-1053](https://opensupplyhub.atlassian.net/browse/OSDEV-1053) - Updated email notification about the claim submission.
```
Comment (Innavin369) [inline doc/release/RELEASE-NOTES.md:40]: The task description should be in the next release section. Am I right?

Comment (mazursasha1990) [reply doc/release/RELEASE-NOTES.md:40]: If it is merged today and successfully tested, these changes will be included in release 1.16.

**Review by Innavin369 — APPROVED** [review]
LGTM

**Pull request merged**